CHANGELOG.md

```diff
@@ -0,0 +1,162 @@
+
+[@crsaracco] has implemented basic support to run druid on bare-metal X11!
+
+While still incomplete this lays the foundation for running druid on Linux without relying on GTK.
+
+#### Mostly complete Wasm backend for druid-shell. ([#759])
+
+[@elrnv] continued the work of [@tedsta] and implemented a mostly complete Wasm backend and enabled all druid example to [run in the browser](https://elrnv.github.io/druid-wasm-examples/).
+
+While some features like the clipboard, menus or file dialogs are not yet available, all fundamental features are there.
+
+### Added
+
+- `TextBox` can receive `EditAction` commands. ([#814] by [@cmyr])
+- `Split::min_splitter_area(f64)` to add padding around the split bar. ([#738] by [@xStorm])
+- Published `druid::text` module. ([#816] by [@cmyr])
+- Basic X11 backend for druid-shell. ([#599] by [@crsaracco])
+- `InternalEvent::MouseLeave` signalling the cursor leaved the window. ([#821] by [@teddemunnik])
+- `children_changed` now always includes layout and paint request. ([#839] by [@xStorm])
+- Mostly complete Wasm backend for druid-shell. ([#759] by [@elrnv])
+- `UpdateCtx::submit_command`. ([#855] by [@cmyr])
+- `request_paint_rect` for partial invalidation. ([#817] by [@jneem])
+- Window title can be any `LabelText` (such as a simple `String`). ([#869] by [@cmyr])
+- `Label::with_font` and `set_font`. ([#785] by [@thecodewarrior])
+
+### Changed
+
+- Renamed `Split` constructors to `Split::rows` and `columns`. ([#738] by [@xStorm])
+- `Split::splitter_size` no longer includes padding. ([#738] by [@xStorm])
+- Renamed `Event::MouseMoved` to `MouseMove`. ([#825] by [@teddemunnik])
+- `has_focus` no longer returns false positives. ([#819] by [@xStorm])
+- `Event::Internal(InternalEvent)` now bundles all internal events. ([#833] by [@xStorm])
+- `WidgetPod::set_layout_rect` now requires `LayoutCtx`, data and `Env`. ([#841] by [@xStorm])
+- `request_timer` now uses `Duration` instead of `Instant`. ([#847] by [@finnerale])
+
+### Deprecated
+
+- Nothing
+
+### Removed
+
+- The optional GTK feature for non-Linux platforms. ([#611] by [@pyroxymat])
+
+### Fixed
+
+- GTK: Use the system locale. ([#798] by [@finnerale])
+- GTK: Actually close windows ([#797] by [@finnerale])
+- Windows: Respect the minimum window size. ([#727] by [@teddemunnik])
+- Windows: Respect resizability. ([#712] by [@teddemunnik])
+- Windows: Capture mouse for drag actions. ([#695] by [@teddemunnik])
+- Start focus cycling from non-registered-for-focus widgets. ([#819] by [@xStorm])
+- Propagate `Event::FocusChanged` to focus gaining widgets as well. ([#819] by [@xStorm])
+- GTK: Prevent crashing on pop-ups. ([#837] by [@finnerale])
+- Keep hot state  consistent with mouse position. ([#841] by [@xStorm])
+- Open file menu item works again. ([#851] by [@kindlychung])
+- Supply correct `LifeCycleCtx` to `Event::FocusChanged`. ([#878] by [@cmyr])
+- Windows: Termiate app when all windows have closed. ([#763] by [@xStorm])
```

Comment:
This same #763 ended up containing a macOS fix too. Perhaps add the following line:
```markdown
- macOS: `Application::quit` now quits the run loop instead of killing the process. ([#763] by [@xStrom])
```